src/lib/components/layout/CardList/CardList.jsx

```diff
@@ -19,7 +20,7 @@ const CardList = (props) => {
       className={styles.root}
       {...other}
     >
-      {React.Children.map(children, (child) => {
+      {flattenChildren(children).map((child) => {
```

Comment:
Fragments are not traversed with `React.children.map()` [1][2] so I'm using the `react-keyed-flatten-children` bundle to flatten the children to a one-dimensional array. This is necessary to be able to pass props to children of these layouts.

[1] https://reactjs.org/docs/react-api.html#reactchildrenmap
[2] https://github.com/reactjs/rfcs/pull/61#issuecomment-584402735